cowtown2023/src/main/java/com/team2357/frc2023/subsystems/ShooterSubsystem.java

```diff
@@ -3,17 +3,42 @@
 import com.revrobotics.CANSparkMax;
 import com.revrobotics.CANSparkMaxLowLevel.MotorType;
 import com.team2357.frc2023.Constants;
+import com.team2357.frc2023.Constants.SHOOTER;
+import com.team2357.lib.subsystems.LimelightSubsystem;
+import com.team2357.lib.util.RobotMath;
 
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 
 public class ShooterSubsystem extends SubsystemBase {
 
     private CANSparkMax m_topShooterMotor;
     private CANSparkMax m_bottomShooterMotor;
+    private SHOOTER m_config;
+    private boolean m_isClosedLoopEnabled;
+    private static final int m_neoMaxRPM = 5676;
     
     public ShooterSubsystem() {
         m_topShooterMotor = new CANSparkMax(Constants.CAN_ID.TOP_SHOOTER_MOTOR_ID, MotorType.kBrushless);
         m_bottomShooterMotor = new CANSparkMax(Constants.CAN_ID.BOTTOM_SHOOTER_MOTOR_ID, MotorType.kBrushless);
+        m_isClosedLoopEnabled = false;
+    }
+
+    public void configure(SHOOTER config){
+        m_config = config;
+
+        m_topShooterMotor.setInverted(m_config.TOP_MOTOR_INVERTED);
```

Comment:
These should all be like this `Constants.SHOOTER.TOP_MOTOR_INVERTED` instead of this `m_config.TOP_MOTOR_INVERTED`